From 064f66379586b754b342269cfdaee1caefcaba95 Mon Sep 17 00:00:00 2001
From: RubenKelevra <ruben@freifunk-nrw.de>
Date: Sat, 17 Jun 2017 23:25:24 +0200
Subject: [PATCH] add all wifi tweaks and reenable >ht20 modes

---
 .../luasrc/lib/gluon/upgrade/200-wireless          | 22 +++++++++++++++++++---
 1 file changed, 19 insertions(+), 3 deletions(-)

diff --git a/package/gluon-core/luasrc/lib/gluon/upgrade/200-wireless b/package/gluon-core/luasrc/lib/gluon/upgrade/200-wireless
index ba844950..a2f4521b 100755
--- a/package/gluon-core/luasrc/lib/gluon/upgrade/200-wireless
+++ b/package/gluon-core/luasrc/lib/gluon/upgrade/200-wireless
@@ -82,7 +82,6 @@ local function configure_ibss(config, radio, index, suffix, disabled)
       ssid = config.ssid,
       bssid = config.bssid,
       macaddr = macaddr,
-      mcast_rate = config.mcast_rate,
       ifname = suffix and 'ibss' .. suffix,
       disabled = disabled,
     }
@@ -121,7 +120,6 @@ local function configure_mesh(config, radio, index, suffix, disabled)
       mesh_id = config.id,
       mesh_fwding = false,
       macaddr = macaddr,
-      mcast_rate = config.mcast_rate,
       ifname = suffix and 'mesh' .. suffix,
       disabled = disabled,
       macfilter = macfilter,
@@ -159,9 +157,27 @@ local function configure_radio(radio, index, config)
 
   uci:delete('wireless', radio, 'disabled')
 
+  local htmode = uci:get('wireless', radio, 'htmode')
+  if htmode ~= 'VHT80' then --enable VHT80 on AC-Devices
+    uci:set('wireless', radio, 'htmode', config.htmode)
+  else
+    if config.htmode == 'HT40-' then --disable VHT80 if second channel is below...
+      uci:set('wireless', radio, 'htmode', config.htmode)
+    end
+  end
+
   uci:set('wireless', radio, 'channel', channel)
-  uci:set('wireless', radio, 'htmode', 'HT20')
+  uci:set('wireless', radio, 'noscan', '1')
   uci:set('wireless', radio, 'country', site.regdom)
+  uci:set('wireless', radio, 'iapp_interface', 'bat0')
+  if hwmode == '11g' then -- 5GHz Wifi does not need this tweaks
+    uci:set('wireless', radio, 'beacon_rate', '60')
+  end
+  uci:set('wireless', radio, 'no_probe_resp_if_max_sta', '1')
+  uci:set('wireless', radio, 'multicast_to_unicast', '1')
+  uci:set('wireless', radio, 'broadcast_deauth', '0')
+  uci:set('wireless', radio, 'proxy_arp', '1')
+  uci:set('wireless', radio, 'na_mcast_to_ucast', '1')
 
   if config.supported_rates then
     uci:set_list('wireless', radio, 'supported_rates', config.supported_rates)
-- 
2.13.1

