From 0530887f9db48870649667bf6ddd205a64625997 Mon Sep 17 00:00:00 2001
From: RubenKelevra <ruben@vfn-nrw.de>
Date: Mon, 18 Jul 2016 15:31:30 +0200
Subject: [PATCH] add routing repo update

---
 modules | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/modules b/modules
index 3c83278..9b56c10 100644
--- a/modules
+++ b/modules
@@ -12,7 +12,7 @@ PACKAGES_GLUON_COMMIT=4e2785a34a6a90c4c377431738d33e047e503777
 PACKAGES_GLUON_BRANCH=v2016.1.x
 
 PACKAGES_ROUTING_REPO=git://github.com/openwrt-routing/packages.git
-PACKAGES_ROUTING_COMMIT=ae65d4fe027592652376f8dbd3ff2ef37f5a84bc
+PACKAGES_ROUTING_COMMIT=a4eae82c155079a4372e4b910ec733f77288b717
 
 PACKAGES_LUCI_REPO=git://github.com/openwrt/luci.git
 PACKAGES_LUCI_COMMIT=cdcdfd2594634804ab09dc8105e46116edce0cd6
-- 
2.9.0

