From a70d489ba86cce40e45fcd7460b58d05310178e6 Mon Sep 17 00:00:00 2001
From: RubenKelevra <ruben@freifunk-nrw.de>
Date: Sat, 17 Jun 2017 23:54:53 +0200
Subject: [PATCH] update gluon-packages

---
 modules | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/modules b/modules
index c3a0802d..6946b76a 100644
--- a/modules
+++ b/modules
@@ -9,7 +9,7 @@ PACKAGES_OPENWRT_BRANCH=lede-17.01
 PACKAGES_OPENWRT_COMMIT=8844d7e2d4f717898c55c6345ad3b43fca52c440
 
 PACKAGES_GLUON_REPO=https://github.com/freifunk-gluon/packages.git
-PACKAGES_GLUON_COMMIT=71823713c0e9451d1cd459cb10309f468188eb6e
+PACKAGES_GLUON_COMMIT=1a8fd4ba87d670b759fc51bb1dc37808e5213be4
 
 PACKAGES_ROUTING_REPO=https://github.com/openwrt-routing/packages.git
 PACKAGES_ROUTING_COMMIT=8d9d70510b2c86f7503962308846ec874f0eb39f
-- 
2.13.1

