From 10c1cdb96ccae22042f87ae181b2ce9981b4f816 Mon Sep 17 00:00:00 2001
From: RubenKelevra <ruben@freifunk-nrw.de>
Date: Sat, 17 Jun 2017 20:47:18 +0200
Subject: [PATCH] remove respondd dependency from fastd-makefile

---
 package/gluon-mesh-vpn-fastd/Makefile | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/package/gluon-mesh-vpn-fastd/Makefile b/package/gluon-mesh-vpn-fastd/Makefile
index 2a1cc067..938337f7 100644
--- a/package/gluon-mesh-vpn-fastd/Makefile
+++ b/package/gluon-mesh-vpn-fastd/Makefile
@@ -4,7 +4,6 @@ PKG_NAME:=gluon-mesh-vpn-fastd
 PKG_VERSION:=3
 
 PKG_BUILD_DIR := $(BUILD_DIR)/$(PKG_NAME)
-PKG_BUILD_DEPENDS := respondd
 
 include ../gluon.mk
 
@@ -29,8 +28,6 @@ endef
 define Package/gluon-mesh-vpn-fastd/install
 	$(CP) ./files/* $(1)/
 	$(CP) $(PKG_BUILD_DIR)/luadest/* $(1)/
-	$(INSTALL_DIR) $(1)/lib/gluon/respondd
-	$(CP) $(PKG_BUILD_DIR)/respondd.so $(1)/lib/gluon/respondd/mesh-vpn-fastd.so
 endef
 
 define Package/gluon-mesh-vpn-fastd/postinst
-- 
2.13.1

