From 6aaa86f733a50705cc8fffefc1cc2b8ac0efb2c5 Mon Sep 17 00:00:00 2001
From: RubenKelevra <ruben@freifunk-nrw.de>
Date: Tue, 19 Jul 2016 02:34:07 +0200
Subject: [PATCH] remove respondd dependency from fastd-makefile

---
 package/gluon-mesh-vpn-fastd/Makefile | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/package/gluon-mesh-vpn-fastd/Makefile b/package/gluon-mesh-vpn-fastd/Makefile
index bb33257..c457765 100644
--- a/package/gluon-mesh-vpn-fastd/Makefile
+++ b/package/gluon-mesh-vpn-fastd/Makefile
@@ -4,7 +4,6 @@ PKG_NAME:=gluon-mesh-vpn-fastd
 PKG_VERSION:=3
 
 PKG_BUILD_DIR := $(BUILD_DIR)/$(PKG_NAME)
-PKG_BUILD_DEPENDS := respondd
 
 include $(GLUONDIR)/include/package.mk
 
@@ -22,9 +21,6 @@ endef
 
 define Package/gluon-mesh-vpn-fastd/install
 	$(CP) ./files/* $(1)/
-
-	$(INSTALL_DIR) $(1)/lib/gluon/respondd
-	$(CP) $(PKG_BUILD_DIR)/respondd.so $(1)/lib/gluon/respondd/mesh-vpn-fastd.so
 endef
 
 define Package/gluon-mesh-vpn-fastd/postinst
-- 
2.9.0

