From ea9fdaf90c413b4c3746d4f5406809f9e13d0108 Mon Sep 17 00:00:00 2001
From: RubenKelevra <ruben@vfn-nrw.de>
Date: Sat, 14 May 2016 21:57:22 +0200
Subject: [PATCH] ipq806x: revert upstream change of KDIR_TMP definition

---
 target/linux/ipq806x/image/Makefile | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/target/linux/ipq806x/image/Makefile b/target/linux/ipq806x/image/Makefile
index 45d0d4d..40aa8ea 100644
--- a/target/linux/ipq806x/image/Makefile
+++ b/target/linux/ipq806x/image/Makefile
@@ -3,6 +3,8 @@
 include $(TOPDIR)/rules.mk
 include $(INCLUDE_DIR)/image.mk
 
+KDIR_TMP=$(KDIR)/tmp
+
 UBIFS_OPTS = -m 2048 -e 124KiB -c 4096 -U -F
 UBINIZE_OPTS = -m 2048 -p 128KiB
 
-- 
2.8.2

